config/locales/en.yml

```diff
@@ -3036,6 +3036,16 @@ en:
           helper_heading: Where do I find how much property taxes I paid?
           label: Property taxes paid in %{filing_year}
           title: Enter the amount of property taxes paid to %{municipality} for your main home in %{filing_year}.
+      nj_homeowner_property_tax_worksheet:
+        edit:
+          anchor_homestead_helper_description: If you received an ANCHOR Benefit, the benefit amount does not affect the amount to report on this line.
+          anchor_homestead_helper_heading: ANCHOR or Homestead benefits?
+          helper_1_html: Good news – we think you might qualify for a credit or deduction. </br></br> The next step is to fill in this <a href="https://www.nj.gov/treasury/taxation/pdf/current/1040_instr_pages/1040i_32.pdf" target="_blank" rel="noopener nofollow">worksheet</a>. </br></br> It will ask you about where you lived and for how long. If you moved, for example, it will have space for you to list every place where you lived. </br></br> Once you fill in the worksheet, come back to this page.
```

Comment:
[pebble] this seems like it should be multiple translation keys that are each tagged as a `<p>` in the erb file markup, rather than one string with breaks.

Reply:
same for tenant